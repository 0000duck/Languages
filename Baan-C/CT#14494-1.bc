|
| https://notepad-plus-plus.org/community/topic/14494/functionlist-classrange-question
|

declaration:                | a section     --> branch
    func2(10.09, "STR")

before.program:             | a section     --> branch
    func1(10)

functions:                  | a section     --> branch

function func1(long var1)   | a function    --> leave
{
    | Do Something 1.
}

function string func2(      | a function    --> leave
            double var2,
            string var2_1
            )
{
    | Do Something 2.
}
