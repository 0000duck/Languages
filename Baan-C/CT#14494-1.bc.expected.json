{
    "root":"CT#14494-1.bc",
    "branches":[
        {
            "name":"functions",
            "leaves":[
                "func1",
                "func2",
            ]
        }
    ],
    "leaves":[
        "declaration",
        "before.program"
    ]
}